src/XeroResourceOwner.php

```diff
@@ -54,7 +80,27 @@ public static function fromJWT($token)
         $self->email = $decoded->email;
         $self->given_name = $decoded->given_name;
         $self->family_name = $decoded->family_name;
+        $self->sub = $decoded->sub;
+        $self->exp = $decoded->exp;
+        $self->auth_time = $decoded->auth_time;
+        $self->iat = $decoded->iat;
+        $self->aud = $decoded->aud;
 
         return $self;
     }
+
+    /**
+     * Provide camelCase access to the underlying properties.
+     *
+     * @param $name
+     * @return mixed
+     */
+    public function __get($name)
```

Comment:
Does a utility function really belong in this class?

Reply:
I don't know. I wanted to deal with the claims as properties in cameCase to be a bit more PSR consistent. The properties were in snake case, as that is how the JWT presents it. I saw the utility function as a bridge between the JWT and PSR worlds.

They could be pulled in as traits or an abstract, and that can be done through refactoring. Or maybe you are suggesting this class should remain as a "JWT value object" that looks to be much closer to the JWT? Happy to go with whatever your line of thinking is.

Reply:
Happy to remove this utility function.

What are your thoughts on making the properties camelCase instead? That would be consistent with the xeroTenant class.

Reply:
Looking at the official generated PHP SDK supplied by Xero, they seem to have chosen snake_case for the model property names. We we the SDK supports something like this through the identity API:

    $connection->tenant_name;
    $connection->getTenantName();

Personally, I generate the API SDK code with camelCase properties, so get this:

    $connection->tenantName;
    $connection->getTenantName();

which is closer to the identity endpoint model in this provider:

    $xeroTenant->tenantName;
    // But no getter method.

It is strange that they have used snake_case in the generated code when even the API payload used camelCase. I'm just mentioning this because it can be a little confusing to peoeple when first using this provider along with the Xero SDK, as the (provider) XeroTenant model, and the (SDK) Connection model are essentially the same thing from the same API endpoint, but sue difference letter case properties. However, if xeroTenant had getter methods too, then that coud be an identical way to access the properties of both objects the same consistent way.